tests/simple/variables.wast

```diff
@@ -19,8 +19,8 @@ init_locals < i32 > 0 : < i32 > 0 : < i32 > 0 : .Stack
 
 ;; Test globals
 
-init_global 0 < i32 > 0
-init_global 1 < i32 > 0
+init_global 0 0
+init_global 1 1
```

Comment:
I think this should be something like:

```
init_global 0 0
(global.set 0 (i32.const 0))
init_global 1 1
(global.set 1 (i32.const 0))
```

Reply:
added tests, implemented folded version and passed local check.